fair_mango/dataset/dataset.py

```diff
@@ -257,18 +600,60 @@ def get_predicted_target_for_all_groups(self) -> list[dict]:
 
     def get_predicted_target_for_one_group(
         self, sensitive: Sequence[str]
-    ) -> pd.DataFrame:
-        """Retrieve predicted target for a specific group
+    ) -> pd.Series | pd.DataFrame:
```

Comment:
same here. Have you created a task for the squeeze ?

Reply:
I removed `.squeeze()`. Now the function always returns `pandas.DataFrame`.